packages/playwright/src/executors/playwright/playwright.ts

```diff
@@ -0,0 +1,179 @@
+import { exec, spawn } from 'child_process';
+import { ExecutorContext, getPackageManagerCommand } from '@nx/devkit';
+
+export async function playwrightExecutor(
+  options: PlaywrightExecutorSchema,
+  context: ExecutorContext
+) {
+  const projectRoot =
+    context.projectGraph?.nodes?.[context?.projectName]?.data?.root;
+
+  if (!projectRoot) {
+    throw new Error(
+      `Unable to find the Project Root for ${context.projectName}. Is it set in the project.json?`
+    );
+  }
+  const args = createArgs(options);
+  const p = runPlaywright(args, projectRoot);
```

Comment:
holding off on adding a 'devServerTarget' feature since playwright does that natively with their webserver config options.